gui/app/modes/PressureAssistMode.qml

```diff
@@ -1,14 +1,229 @@
-import QtQuick 2.0
-
+import QtQuick 2.11
+import QtQuick.Layouts 1.3
+import QtQuick.Controls 2.4
+import Respira 1.0
 import "../controls"
 import ".."
 
-// TODO: Implement mode
+// TODO: Common this with CommandPressureMode.qml.
```

Comment:
Please rebase and re-copy-paste (I know, I know...), some values here have changed in https://github.com/RespiraWorks/VentilatorSoftware/pull/518

Reply:
@jkff DM'ed me and said, merge as-is, he'll clean it up.